tutorclickhouse/plugin.py

```diff
@@ -48,6 +48,8 @@
         # Prefix your setting names with 'CLICKHOUSE_'.
         ("CLICKHOUSE_ADMIN_USER", "ch_admin"),
         ("CLICKHOUSE_ADMIN_PASSWORD", "{{ 24|random_string }}"),
+        ("CLICKHOUSE_SECURE_CONNECTION", False),
```

Comment:
why is False the default?

Reply:
If you are running clickhouse (in k8s), we can skip the TLS validation (we can't also). The default clickhouse port is 9000